client/galaxy/scripts/components/RuleCollectionBuilder.vue

```diff
@@ -1029,7 +1029,7 @@ export default {
         }
         return {
             rules: rules,
-            colHeadersPerRule: [],
+            // colHeadersPerRule: [],
```

Comment:
We probably should just remove this line.

Reply:
Yup